evaluate2.sh

```diff
@@ -149,9 +151,13 @@ for fork in "$@"; do
 
     # Linux platform
     # prepend this with numactl --physcpubind=0-7 for running it only with 8 cores
-    numactl --physcpubind=0-7 hyperfine $HYPERFINE_OPTS "./calculate_average_$fork.sh 2>&1"
-  else
-    hyperfine $HYPERFINE_OPTS "./calculate_average_$fork.sh 2>&1"
+    numactl --physcpubind=0-7 hyperfine $HYPERFINE_OPTS "timeout -v $RUN_TIME_LIMIT ./calculate_average_$fork.sh 2>&1"
```

Comment:
I get 
```
timeout: invalid option -- 'v'
Try 'timeout --help' for more information.
```
It works without -v

Reply:
I guess I have outdated version, I see -v here https://man7.org/linux/man-pages/man1/timeout.1.html